assignments.js

```diff
@@ -0,0 +1,53 @@
+/* Fibonacci series using map Assignment question-1 */
+
+export function fibonacciSeries(number) {
+  const fib = [0, 1];
+  for (let i = 2; i <= number; i++) {
+    fib[i] = fib[i - 1] + fib[i - 2];
+  }
+  return fib;
+}
+export function mappingIndex(numbers) {
+  const maxNumber = Math.max.apply(null, numbers);
+  const fibSeries = fibonacciSeries(maxNumber);
+  return numbers.map((number) => fibSeries[number]);
+}
+
+/* Converting average temperature into fahrenite Assignment Question-2 */
+
+export const result2 = (poiArray) =>
+  poiArray.map((item) => Math.floor((item.averageTemperature * 9) / 5 + 32));
+
+/* Additional Assignment Question-1 */
+
+export function createCutOff(cutOffValue) {
+  return (number) => number < cutOffValue;
+}
+
+/* Additional Assignment Question-2 */
+export const result4 = (strings) =>
+  strings.map((str) => str.replace("CraftCode", "CodeCraft"));
+
+/* Additional Assignment Question-3 */
+
+export const res3 = (purchases) =>

```

Comment:
Why we are naming functions as res3? We need to be very careful before we name a consturct in programming. Function should ideally begin with verb, or sound like accessing somethig or transforming something.